src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -395,6 +403,12 @@ private function addAssetsSection(ArrayNodeDefinition $rootNode)
                                         ->prototype('scalar')->end()
                                     ->end()
                                 ->end()
+                                ->validate()
+                                    ->ifTrue(function ($v) {
+                                        return (null !== $v['version_strategy'] && null !== $v['version']);
+                                    })
+                                    ->thenInvalid('You cannot use version_strategy and version settings in same package.')
```

Comment:
`You cannot use both "version_strategy" and "version" at the same time under "assets" for the "%s" package.`